### ISSUE TITLE
Green and Teal primary buttons fail contrast accessibility in dark theme

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#1354]
**Package** lib-react-components

**Is your feature request related to a problem? Please describe.**
The contrast does not meet accessibility contrast standards of 4.5:1

**Additional context**

Green button with dark theme:
<img width="108" alt="Screen Shot 2019-12-04 at 2 48 28 PM" src="https://user-images.githubusercontent.com/5016731/70180267-46b67900-16a5-11ea-8b81-6a0520e80dec.png">
Fails contrast test between the green and background color with 3.04

Teal button using dark theme:
<img width="110" alt="Screen Shot 2019-12-04 at 2 48 18 PM" src="https://user-images.githubusercontent.com/5016731/70180312-5f269380-16a5-11ea-95e2-4a62b8d22834.png">
Fails contrast test between the teal and background color with 3.55

**Comment by beckyrother** [comment]
Outlined buttons in dark theme should have white text with different colored outlines:
![image](https://user-images.githubusercontent.com/1392130/70182817-35bc3680-16aa-11ea-99eb-af3276556c87.png)

Good example: https://projects.invisionapp.com/d/main#/console/12923997/283335200/preview
